mir-module/src/test/java/org/mycore/mir/editor/MIREditorUtilsTest.java

```diff
@@ -0,0 +1,71 @@
+package org.mycore.mir.editor;
+
+import org.apache.logging.log4j.LogManager;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.safety.Whitelist;
+import org.jsoup.select.Elements;
+import org.junit.Assert;
+import org.junit.Test;
+import org.mycore.common.MCRTestCase;
+
+import java.util.Map;
+
+import static org.junit.Assert.*;
```

Comment:
Codacy found an issue: [Using the '.*' form of import should be avoided - org.junit.Assert.*.](https://app.codacy.com/manual/MyCoRe/mir/pullRequest?prid=6223128)